### DISCUSSION TITLE
v0.31.2

### DISCUSSION THREAD

**Discussion by borg323** [LeelaChessZero/lc0#2080] [General]
In this version:
* Updated the WDL_mu centipawn fallback.
* Fix for build issues with newer Linux c++ libraries.
* Fix for an XLA Mish bug.
* Minor README.md update.

<hr /><em>This discussion was created from the release <a href='https://github.com/LeelaChessZero/lc0/releases/tag/v0.31.2'>v0.31.2</a>.</em>

**Comment by Jubilao3** [comment]
Fritz 19, analysis mode with Lc0 v0.31.2. The computer hangs and the following message appears:

"Lc0 v0.31.2"caused an exception.
This is a problem in the engine.

Win 10.0 x64
65366 MB
13th Gen Intel(R) Core(TM) i9-13900KF
N=1/P=1/C=24/L032/G=
NVIDIA GeForce RTX 4090
NVIDIA GeForce RTX 4090
NVIDIA GeForce RTX 4090
NVIDIA GeForce RTX 4090

**Reply by borg323** [reply]
Does Lc0 run outside Fritz?

**Reply by borg323** [reply]
You can follow the procedure outlined in https://github.com/LeelaChessZero/lc0/discussions/1669#discussioncomment-2798404

**Reply by Jubilao3** [reply]
Same problem in Nibbler. But I'm trying MinibatchSize=16, and so far it's working fine. Thanks for your reply.

**Reply by borg323** [reply]
Did a previous lc0 version work for you? Did anything else change in your configuration?

**Reply by Jubilao3** [reply]
I had the same problem with the previous version, but I gave up and left it. But I really like Lc0 and now I have dared to expose the problem. I think it has been solved with MinibathSize=16.

**Reply by faniti** [reply]
Hello,

**Comment by lbgkm** [comment]
As I experienced the Lc0 ver.0.31.1 and ver.0.31.2 are the most faster binaries.
I maid tests with 0.31.1 and some NVIDIA RTX 4090 GPUs as follows:
TC was 1 min + 2sec/move, startpos. number 100 from standard chess opening positions, 
with altered colors (200 games/test)
and I used the TCEC 25 params with BT4-1024x15x32h-swa-607750 NET for Lc0 
and 64 x 3600MHz (SMT) CPU, 2048 MB hash for Stockfish
Test with binay Lc0 ver.0.31.1
1 x RTX 4090 and Stockfish dev.24.10.12 result 1 : 8 Elo diff. ~ +12 Elo for Stockfish
2 x RTX 4090 and Stockfish dev.24.10.12 result 3 : 4 Elo diff. ~ +2 Elo for Stockfish
3 x RTX 4090 and Stockfish dev.24.10.12 result 2 : 4 Elo diff. ~ +3 Elo for Stockfish
Note
1,It would be important to use the standard chess opening positions not for real tests but for TCEC
competitions also, because the positions resulted by bad or pointless play give too big benefit 
for the tactical engines as Stockfish is!
2,TCEC GPU server is more powerfull than 3 x RTX 4090 gamer GPUs....
Regards
corres

**Comment by lbgkm** [comment]
I repeated the above test with RTX 3080 TI (EVGA, Its power is about the same as an RTX 3090)
The results are:
Stockfish dev.24.10.12.  :  Lc0-ver.0.31.1  =  12 : 1  Elo diff. ~ +19 Elo for Stockfish

Note
The power of RTX 3080Ti is about the half of RTX 4090.

**Comment by lbgkm** [comment]
1,Based on my experience LC0-v0.31.1 + BT4-1024x15x32h-swa-6077500 is stronger 
  than LC0-v0.32(Borg) + BT4-1024x15x32-swa-6147500-policytune-332
2,Using of UHO-type opening positions is very detrimental for LC0.
  Why accept LC0 development team its using?
3, In my opinion TCEC is a "vision bakery" only without any advantage for chess players.
corres
